### ISSUE TITLE
Error creating pipeline: Create pipeline failed: Invalid input error: Unsupported argo version. Expected: tekton.dev/v1beta1. Received: tekton.dev/v1

### ISSUE THREAD

**Issue by adelton** [RedHatQuickCourses/rhods-pipelines#23]
Following https://redhatquickcourses.github.io/rhods-pipelines/rhods-pipelines/1.33/chapter1/kfp.html, when I build the `coin-toss.yaml` with `python3 coin-toss.py` in a launcher terminal of an OpenShift AI 2.8.0's `fraud-detection-workbench` workbench created in https://redhatquickcourses.github.io/rhods-pipelines/rhods-pipelines/1.33/chapter1/elyra-pipelines.html, the produced YAML start with
```
(app-root) (app-root) head coin-toss.yaml 
apiVersion: tekton.dev/v1
kind: PipelineRun
metadata:
  name: conditional-execution-pipeline
  annotations:
    tekton.dev/output_artifacts: '{"flip-coin": [{"key": "artifacts/$PIPELINERUN/flip-coin/Output.tgz",
```

When I then import that YAML as a `coin-toss-pipeline`, I get
> Danger alert:Error creating pipeline
> Error creating pipeline: Create pipeline failed: Invalid input error: Unsupported argo version. Expected: tekton.dev/v1beta1. Received: tekton.dev/v1

![Screenshot_2024-04-04_15-55-59](https://github.com/RedHatQuickCourses/rhods-pipelines/assets/2536912/2260a7e9-a7af-4654-9f86-c05f53d602b0)

**Comment by rsriniva** [comment]
@adelton - I fixed this. will submit PR and close this soon. Thank you for reporting this. The fix is to downgrade the version of kfp-tekton to 1.5.X. The coin toss example works for me now. will push to main soon...

**Comment by rsriniva** [comment]
fixed in #26
